account/account_test.go

```diff
@@ -530,6 +532,51 @@ func TestTransactionHashDeclare(t *testing.T) {
 	require.Equal(t, expectedHash.String(), hash.String(), "TransactionHashDeclare not what expected")
 }
 
+func TestWaitForTransactionReceiptMOCK(t *testing.T) {
```

Comment:
Do we need tests for polling ?

Reply:
I've added a separate test for polling using devnet